### PR TITLE
Create Manufacturer_smartron_rimo02a

### PR DESCRIPTION
**Pull request by DevendarReddyDarukumalli**

### CODE REVIEW

File: Manufacturer_smartron_rimo02a [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<manufacturer id="Manufacturer">
+        <DeviceCode>
```
Comment (gwolf2u) [inline Manufacturer_smartron_rimo02a]: This needs to be your device codename 
rimo02a

File: Manufacturer_smartron_rimo02a [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            <maintainer>DevendarReddyD</maintainer>
+            <devicename>SmartronSRT</devicename>
+            <filename>crDroidAndroid-9.0-20190324-rimo02a-v5.2</filename>
+            <buildtype>Monthly</buildtype>
+            <download>http://bit.ly/2JPWU1R</download>
+            <changelog>ChangelogLink</changelog>
```
Comment (gwolf2u) [inline Manufacturer_smartron_rimo02a]: This can be blank (delete changelog link)

File: Manufacturer_smartron_rimo02a [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            <gapps>https://opengapps.org/</gapps>
+            <forum>https://forum.xda-developers.com/member.php?u=9815186</forum>
```
Comment (gwolf2u) [inline Manufacturer_smartron_rimo02a]: This should be device forum link not your username

File: Manufacturer_smartron_rimo02a [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            <firmware>FirmwareLink</firmware>
```
Comment (gwolf2u) [inline Manufacturer_smartron_rimo02a]: If you don't have firmware, modem, bootloader, recovery links, then just delete it
Those tags are optional

File: Manufacturer_smartron_rimo02a [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+            <modem>ModemLink</modem>
+            <bootloader>BootloaderLink</bootloader>
+            <recovery>RecoveryLink</recovery>
+            <paypal>paypal.me/DevendarReddyD</paypal>
+            <telegram>https://t.me/DevendarReddy</telegram>
+        </DeviceCode>
```
Comment (gwolf2u) [inline Manufacturer_smartron_rimo02a]: This should also be your device codename

**Review by gwolf2u — CHANGES_REQUESTED** [review]
See my review

**Comment by gwolf2u** [comment]
info from file Manufacturer_smartron_rimo02a supposed to be added in update.xml
please recommit

**Comment by DevendarReddyDarukumalli** [comment]
added in update.xml

**Comment by gwolf2u** [comment]
please open a new PR

**Pull request closed without merging**